packages/Set/ValueObject/SetList.php

```diff
@@ -117,6 +117,18 @@ final class SetList implements SetListInterface
      */
     public const PRIVATIZATION = __DIR__ . '/../../../config/set/privatization.php';
 
+    /**
+     * @var string
+     */
+    public const TYPE_DECLARATION_RETURN = __DIR__ . '/../../../config/set/return-type-declaration.php';
+    /**
+     * @var string
+     */
+    public const TYPE_DECLARATION_PARAMETER = __DIR__ . '/../../../config/set/parameter-type-declaration.php';
+    /**
+     * @var string
+     */
+    public const TYPE_DECLARATION_PROPERTY = __DIR__ . '/../../../config/set/property-type-declaration.php';
```

Comment:
Please rename constant to  "X type" declaraions, as its more natual to read and fits file names.